SPECS/perl-Devel-CheckBin/perl-Devel-CheckBin.spec

```diff
@@ -56,6 +56,7 @@ make test
 %changelog
 * Fri Oct 15 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 0.04-14
 - Initial CBL-Mariner import from Fedora 32 (license: MIT).
+- License verified
```

Comment:
We should bump the release number, so that `rpm` is able to disambiguate between the extended and core.